package.json

```diff
@@ -61,7 +60,7 @@
     "smee-client": "^1.2.2"
   },
   "engines": {
-    "node": ">= 8.3.0"
+    "node": ">= 10"
```

Comment:
I don't disagree with the change, but _technically_ you should publish this (or the next change that does get published) as a major bump when doing this.

Reply:
Good call -- I can publish this as 4.0.0.